src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -405,6 +406,54 @@ async Task IDurableOrchestrationClient.TerminateAsync(string instanceId, string
             }
         }
 
+        async Task IDurableOrchestrationClient.SuspendAsync(string instanceId, string reason)
+        {
+            OrchestrationState state = await this.GetOrchestrationInstanceStateAsync(instanceId);
+            if (IsOrchestrationRunning(state))
+            {
+                // TODO: will suspend events target a particular execution ID?
+                state.OrchestrationInstance.ExecutionId = null;
```

Comment:
Good question. I guess we haven't really discussed how suspend interacts with the "continueAsNew" API, which generates new executionIDs. Let's get back to this.

Reply:
Consensus (in another comment in this PR) is that suspend-resume won't use executionIDs.